src/network/service.rs

```diff
@@ -193,7 +198,7 @@ impl ContentService for NodeService {
 
     fn message_subscription<S>(
         &mut self,
-        subscriber: Self::NodeId,
+        _subscriber: Self::NodeId,
```

Comment:
Eh, I've been keeping this so the warning reminds me that we have to add an implementation at some point. Please remove this completely unrelated change :)